<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dico</class>
 <widget class="QMainWindow" name="dico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dico</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label3d">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>24</y>
      <width>231</width>
      <height>375</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>567</x>
      <y>30</y>
      <width>31</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonf">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>390</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttont">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonm">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonw">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>390</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonson">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonsonoff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonrecherche">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>33</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttongoogle">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>64</y>
      <width>521</width>
      <height>357</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(36, 36, 36);</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetR">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>58</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonaccueil">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>25</y>
      <width>63</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonreturn">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>25</y>
      <width>41</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Buttonliste">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>33</y>
      <width>28</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
